Gemfile

```
@@ -66,3 +67,5 @@ gem "devise"
 gem "devise-i18n"
 
 gem "omniauth-github"
+
+gem 'ruby-vips'
```

他の既存項目が主にダブルクォートを使っているみたいなので、揃えた方が良いかなと思いました 👀 

```suggestion
gem "ruby-vips"
```